notebook/static/base/js/utils.js

```diff
@@ -1038,7 +1038,34 @@ define([
             return (order == 1) ? 1 : -1;
         }
     };
+    
+    // source: https://github.com/sindresorhus/pretty-bytes
```

Comment:
I'm OK with this since it's one fairly small function, and it's unlikely to change. But we should include a copy of the license in the comment.

https://github.com/sindresorhus/pretty-bytes/blob/master/license